ceph-releases/ALL/rhel8/daemon-base/__DOCKERFILE_POSTINSTALL_CLEANUP__

```diff
@@ -0,0 +1,5 @@
+echo 'Postinstall cleanup' && \
+ ( yum clean all && \
+   rpmdb --rebuilddb && \
```

Comment:
This `rpmdb --rebuilddb` command fails with the following error:

```
error: failed to replace old database with new database!
error: replace files in /var/lib/rpm with files from /var/lib/rpmrebuilddb.1096 to recover
```

Reply:
Same on fedora 29. I'm wondering if this could come from the overlayfs issue with rename command on that FS.
I think that was resolved on EL7 by using the yum ovl plugin but I might be wrong (I tried dnf-plugin-ovl with no difference).

Reply:
I think we should take this line out for RHEL 8. I've filed https://bugzilla.redhat.com/show_bug.cgi?id=1680124 to track this with the RPM team.